app/src/main/java/com/cornellappdev/android/eatery/ui/components/details/FavoritesToggle.kt

```diff
@@ -0,0 +1,88 @@
+package com.cornellappdev.android.eatery.ui.components.details
+
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
```

Comment:
Android studio's telling me there are an asston of unused imports here, can u run `Optimize Imports` here to remove them? 
![image](https://github.com/user-attachments/assets/928e8488-2a13-41fe-8505-cb624588139c)

Reply:
Done!